desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -117,7 +118,7 @@ def _convert_format(format_dict, inverse=False):
 def guess_format(request):
   file_format = json.loads(request.POST.get('fileFormat', '{}'))
 
-  if file_format['inputFormat'] == 'file':
+  if file_format['inputFormat'] == 'file' or file_format['inputFormat'] == 'localfile' :
     path = urllib_unquote(file_format["path"])
     indexer = MorphlineIndexer(request.user, request.fs)
     if not request.fs.isfile(path):
```

Comment:
looks like error is here (in line 124) function is not going forward after this
<img width="1765" alt="Screenshot 2021-03-24 at 7 07 21 PM" src="https://user-images.githubusercontent.com/36241930/112319524-2c5a4180-8cd4-11eb-903d-7231ce66665d.png">